public/js/index.js

```diff
@@ -2301,12 +2301,23 @@ function initNavbarContentToggle() {
     });
 }
 
+function pasteFilter(e) {
```

Comment:
What does this magicaly function do? Mind adding that to a comment above the function? :)

Reply:
It replaces all tags (substrings on the form <...>) with a space, " " when a user pastes, but I agree there should be a comment.